assembl/static/js/app/views/messageColumnsPanel.js

```diff
@@ -346,10 +348,16 @@ var MessageColumnView = BaseMessageColumnView.extend({
     BaseMessageColumnView.prototype.onRender.apply(this, arguments);
     var that = this,
         header = this.model.get('header'),
+        synthesis_title = this.model.get('synthesis_title'),
         canEdit = Ctx.getCurrentUser().can(Permissions.ADMIN_DISCUSSION),
         renderId = _.clone(this._renderId);
 
     if (this.processIsEnded() || canEdit) {
+      this.messageColumnSynthesisTitle.show(new CKEditorLSField({
```

Comment:
A synthesis title can contain just text and not rich text, so why do you use a rich text editor and then remove rich text in backend using `sanitize_text`, instead of just using a non-rich text field like `editableLSField`? I think for non-rich text inputs, frontend should not submit rich text to backend.

Reply:
I tried to use editableLSField, but this is not the same behavior as the CKEditorLSField. When I click on the field, it was not the correct language I wanted to edit.
I don't well understand this stuff, so if you want to give it a try, please commit in the branch.

Reply:
OK I'm starting to investigate. As is, it cannot work either because CKEditorLSField has no translationData parameter given.